recipe_scrapers/quitoque.py

```diff
@@ -0,0 +1,146 @@
+# mypy: allow-untyped-defs
+
+import requests
+
+from ._abstract import AbstractScraper
+
+
+class QuiToque(AbstractScraper):
+    def __init__(self, url, proxies=None, timeout=None, *args, **kwargs):
+        super().__init__(url=url, proxies=proxies, timeout=timeout, *args, **kwargs)
+        recipe_id = url.split("/")[4]
+        data_url = "https://mgs.quitoque.fr/graphql"
+        data_query = """
+query getRecipe($id: ID!) {
+    recipe(id: $id) {
+        name
+        shortDescription
+        image
+        nutritionalInformations {
+            kiloCalorie
+            fat
+            saturatedFat
+            carbohydrate
+            sugarCarbohydrate
+            fiber
+            protein
+            salt
+        }
+        facets {
+            name
+        }
+        pools {
+            nbPerson
+            cookingModes {
+                cookingTime
+                steps{
+                    description
+                }
+                stacks{
+                    tools{
+                        name
+                    }
+                    cupboardIngredients{
+                        quantity
+                        literalQuantity
+                        product{
+                            name
+                        }
+                    }
+                    ingredients{
+                        literalQuantity
+                        quantity
+                        product{
+                            name
+                        }
+                    }
+                }
+                waitingTime
+            }
+        }
+    }
+}
+        """
+        data_body = {
+            "operationName": "getRecipe",
+            "variables": {"id": recipe_id},
+            "query": data_query,
+        }
+        self.data = requests.post(
+            data_url,
+            json=data_body,
+            proxies=proxies,
+            timeout=timeout,
+        ).json()["data"]
+
+    @classmethod
+    def host(cls):
+        return "quitoque.fr"
+
+    def title(self):
+        return self.data["recipe"]["name"]
+
+    def category(self):
+        categories = ""
+        for category in self.data["recipe"]["facets"]:
+            categories += f'{category["name"]}, '
+        return categories[:-2]
```

Comment:
Perhaps we should use a Python `set` datatype to hold and de-duplicate the categories?

Reply:
This is not a bad idea. That should avoid to return `categories[:-2]`. I fix this.

Reply:
Resolved in [948f289](https://github.com/hhursev/recipe-scrapers/pull/1163/commits/948f289badbd116a4c8fbd3d554e85e4767f9644)